doc/api.md

```diff
@@ -34,49 +34,54 @@ Its constructor gets a number of options:
 Example:
 
 ```javascript
-var domain = "meet.jit.si";
-var options = {
-    roomName: "JitsiMeetAPIExample",
+const domain = 'meet.jit.si';
+const options = {
+    roomName: 'JitsiMeetAPIExample',
     width: 700,
     height: 700,
     parentNode: document.querySelector('#meet')
-}
-var api = new JitsiMeetExternalAPI(domain, options);
+};
+const api = new JitsiMeetExternalAPI(domain, options);
 ```
 
 You can set the initial media devices for the call:
 
 ```javascript
-var domain = "meet.jit.si";
-var options = {
+const domain = 'meet.jit.si';
+const options = {
     ...
     devices: {
         'audioInput': '<deviceLabel>',
         'audioOutput': '<deviceLabel>',
         'videoInput': '<deviceLabel>'
```

Comment:
If we're gonna take shots at styling, the quotes around the keys are unnecessary and, more importantly, inconsistently used within this file.

Reply:
agreed

Reply:
Fixed!